jobs/build/oc_sync/publish-clients-from-payload.sh

```diff
@@ -3,8 +3,9 @@ set -eux
 export MOBY_DISABLE_PIGZ=true
 WORKSPACE=$1
 VERSION=$2
-CLIENT_TYPE=$3
-PULL_SPEC=$4
+RELEASE_TAG=$3
```

Comment:
I did pass the tag as an script argument, but nothing bellow seems to use it (because PULL_SPEC already comes with the tag)

Reply:
huh, good point. i guess i'd prefer to leave it out then, but no strong feelings.

Reply:
removed :heavy_check_mark: